README.md

```diff
@@ -34,20 +34,17 @@ Once the data is received, light client verifies individual cells and calculates
 
 ## Installation
 
-Start by cloning this repo in your local setup:
+Suppose you are already running our [node](https://github.com/availproject/avail). For this setup we will run it in dev mode:
 
-```ssh
-git clone git@github.com:maticnetwork/avail-light.git
+```sh
+data-avail --dev
```

Comment:
Maybe its good recommendation to run in tmp mode also, for a fresh start, like:

`data-avail --dev --tmp`

Reply:
`--dev` implies `--tmp`, so no difference here

Reply:
Cool, lets merge it than.